### PR TITLE
[changelog] Add entry for 6.2.0

### PR DESCRIPTION
**Pull request by olivielpeau**
[skip ci]

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,191 @@
 Release Notes
 =============
 
+6.2.0
+=====
+2019-05-11
```
Comment (koudaiii) [inline CHANGELOG.rst]: 2018 ? 🤔

Comment (olivielpeau) [reply CHANGELOG.rst]: 🤦‍♂️ my bad, let me fix this, thanks @koudaiii!

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,184 @@
...
+- Introduce new docker cpu shares gauge.
+
+- Add ability to configure the namespace in which the resources related to the kubernetes check are created.
+
+- The kubelet check now honors container filtering options
+
+- Adding Datadog Cluster Agent client in Node Agent.
+  Adding support for TLS in the Datadog Cluster Agent API.
+
+- Introduce the flare command for the DCA CLI.
```
Comment (xvello) [inline CHANGELOG.rst]: DCA-only change, can the removed from the agent changelog

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,184 @@
...
+
+- Add a svcmap command in the CLI of the Datadog Cluster Agent. Core components will also be used for the flare.
```
Comment (xvello) [inline CHANGELOG.rst]: ditto

File: CHANGELOG.rst [hunk 4]
```diff
@@ -2,6 +2,184 @@
...
+  `kubernetes.yaml` file, for agents running on the host
+
+- The memory corecheck sends 2 new metrics on Linux: ``system.mem.commit_limit``
+  and ``system.mem.committed_as``
+
+- Added the possibility to filter docker containers by name for log collection.
+
+- Added a support for docker labels to enrich logs metadata.
+
+- Add a `filename` tag to messages with the name of the file being tailed.
```
Comment (xvello) [inline CHANGELOG.rst]: That's logs iirc, the line by itself is not clear :/

File: CHANGELOG.rst (hunk 1 above)
Comment (koudaiii) [reply CHANGELOG.rst]: Thank you 😄 I'm looking forward to 6.2.0 being released 👍 👍 👍

File: CHANGELOG.rst [hunk 5]
```diff
@@ -2,6 +2,184 @@
...
+-----------------
+
+- Removing python PDH code bundled with the agent in favor of code already included
+  in the integrations-core` repository and bundled with datadog_checks_base wheel.
+  This provides a single source of truth for the python PDH logic.
+
+Security Issues
+---------------
+
+- Separate the tokens for the internal API (used for flare, status, metamap) and the external endpoint (used for the tagger of the agent).
```
Comment (xvello) [inline CHANGELOG.rst]: DCA-only too

File: CHANGELOG.rst [hunk 6]
```diff
@@ -2,6 +2,184 @@
...
+  kubernetes_state integration
+
+- Fix case issues in tag extraction for docker/kubernetes container tags and kubernetes host tags
+
+- Fixes initialization of performance counter (Windows) to be able to better cope with missing
+  counter strings, and non-english locales
+
+- Bind the kubelet_tls_verify as an environment variable.
+
+- Kubernetes: remove configmap listing logic, as leader election now uses endpoints
```
Comment (xvello) [inline CHANGELOG.rst]: Changed twice in the release lifecycle: reverted to configmaps for LE

**Pull request merged**